metricbeat/module/kubernetes/state_container/_meta/test/kube-state-metrics.expected

```diff
@@ -1,313 +1,445 @@
 [
-	{
-		"_module": {
-			"namespace": "kube-system",
-			"node": {
-				"name": "minikube"
+    {
+		"RootFields": null,
+		"ModuleFields": null,
+		"MetricSetFields": {
+			"_module": {
```

Comment:
Something is odd here or I don't fully understand it. `_module` etc. comes from the old interface. Why do we still have it with the new interface? I think I miss something here probably related to the generation of the expected files.

Reply:
It also looked weird for me, that's why I pushed it so any of you can check if it was correct. If it gives you some clue, the new expected files where generated with https://github.com/elastic/beats/blob/master/metricbeat/helper/prometheus/ptest/ptest.go#L39 after removing the "version" I wrote of `ptest` to use ReporterV2